### DISCUSSION TITLE
Welcome to ACE3 Discussions!

### DISCUSSION THREAD

**Discussion by jonpas** [acemod/ACE3#8038] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by PabstMirror** [comment]
🍌

**Reply by veteran29** [reply]
🍌

**Reply by diwako** [reply]
🍌

**Reply by Quailsnap** [reply]
🙈

**Comment by Drofseh** [comment]
Welcome to the first ACE3 Discussion Committee meeting, the meeting is now in session.

Our only order of business on the agenda is to elect a Chairman of the Discussion Committee.

We will now take nominations for Committee Chairman.

As the sole Member of the ACE3 Discussion Committee I nominate myself for the position of Chairman of the Discussion Committee.

As the sole Member of the Committee I accept this nomination on behalf of the Committee.

My first order of business as Chairman of the Discussion Committee is to propose that the Committee nominate the song "Yes, We Have No Bananas!" as performed by Louis Prima & His Orchestra to be adopted as the anthem of the Committee, played at the start and finish of each Committee meeting.

Due to the minimal number of Committee Members present I believe a simple vote by roll call is all that will be required, and we will move directly to the vote.

All who are opposed say "Nay".

*silence*

All who is in support say "Yea".

*"Yea"*

The votes have been tallied and, with one vote "Yea" and no votes "Nay", the proposal appears to have unanimous support and is so passed.

Congratulations everyone on the adoption of our new anthem.

Do any other Committee Members have any new business to bring before the Committee?

*silence*

As no one has put forward any new business I declare the Committee meeting to be adjourned following the playing of the anthem.

[!["Yes, We Have No Bananas!"](https://img.youtube.com/vi/8QqkrIDeTeA/0.jpg)](https://www.youtube.com/watch?v=8QqkrIDeTeA)

The Committee is now adjourned.

**Reply by mjc4wilton** [reply]
Hello, this is the ACE3 Discussion Committee Investigation Subcommittee. As founding member of the ACE3 Discussion Committee Investigation Subcommittee I have launched an investigation into whether the ACE3 Discussion Committee is respecting the opportunity to invite comments from the public before and after every meeting as is written in the bylaws. In order to thoroughly investigate all parties involved in this dispute, I require all meeting minutes from the last 2 years from both public and closed door meetings. Failure to comply will result in the reformation of the ACE3 Discussion Committee as is written in the bylaws.
